docs/next/guides/getting-started/setting-options.md

```diff
@@ -14,319 +14,219 @@ tags:
+  cells(row, column) {
+    if (row === 1 && column === 1) {
+      return { readOnly: true };
     }
-
-    return cellProperties;
-  }
+  },
+  width: 'auto',
+  height: 'auto',
+  rowHeaders: true,
+  colHeaders: true,
+  licenseKey: 'non-commercial-and-evaluation'
 });
 ```
+:::
 
-The above notation will result in all TDs being read only, except for first column TDs which will be editable, except for the TD in top left corner which will still be read-only.
-
-## The cascading configuration model
+## `cells` vs `cell`
 
-The cascading configuration model is based on prototypal inheritance.
+In the Options section of the API, you can find two similar options, [cell](@/api/metaSchema.md#cells-2) and [cell](@/api/metaSchema.md#cell). Using one of these options (or both), you can associate the custom properties or overwrite any entire table or column option for a particular cell. Both options allow you to set initial values for specific cells. You can check if the values are propagated by calling the [getCellMeta](@/api/core.md#getcellmeta) method. The method returns an object with all built-in options as well as those added by the developer.
 
-### Constructor
+#### `cell`
 
-Configuration options that are provided using first-level and `updateSettings` method.
+The [cell](@/api/metaSchema.md#cell) option works great for cases when you have to set initial values for your custom properties or change the initial values for the build-in options. Once the changes are propagated to the cells' meta-objects, they can be modified by the Handsontable while it is running. For example, the cell read-only state can be toggled by the context menu.
 
 ```js
-const container = document.querySelector('#example');
+// ...
+cell: [
+  { row: 1, col: 1, readOnly: true }
+],
+// ...
 
-new Handsontable(container, {
-  option: 'value'
-});
+hot.getCellMeta(0, 0).readOnly === false; // By default the option is "false"
+hot.getCellMeta(1, 1).readOnly === true; // the "cell" overwrites the value
+
+// The state can be changed using API or UI e.g.
+hot.setCellMeta(1, 1, 'readOnly', false);
 ```
-  
 
-### Columns
+#### `cells`
 
-Configuration options that are provided using second-level object.
+The [cells](@/api/metaSchema.md#cells-2) option works slightly differently. The `cells` option is a function that invokes the logic in it before the table's rendering cycle. The options returned by that function always overwrite the entire table or column options. Thus, if you implement logic incorrectly, you won't be able to change cells' state from API or UI - as the values will be constantly overwritten by the `cells` function. You can think of this option as an opportunity to transfer the responsibility of keeping the table cells option to your application.
 
-```js
-const container = document.querySelector('#example');
+If you want to stop, overwrite the cell meta object for particular cells, just return an empty object.
```

Comment:
Can't decipher this sentence. Please remix.

Reply:
I rephrased the sentence as https://github.com/handsontable/handsontable/blame/67059fd0a6816041b52137ce3713a9bce3b80d5c/docs/next/guides/getting-started/setting-options.md#L190.